devlib/trace/perf.py

```diff
@@ -65,46 +80,61 @@ class PerfCollector(TraceCollector):
         man perf-stat
     """
 
-    def __init__(self, target,
+    def __init__(self, 
+                 target,
+                 perf_type='perf',
+                 command='stat',
                  events=None,
                  optionstring=None,
+                 report_options=None,
                  labels=None,
                  force_install=False):
         super(PerfCollector, self).__init__(target)
+        self.perf_type =perf_type
```

Comment:
- Fix spacing around `=`.
- Should validate that `pref_type` is one of the allowed values.

Reply:
Thanks, updated.